test/integration/manifests/cilium/cilium-config-hubble.yaml

```diff
@@ -49,7 +49,6 @@ data:
   hubble-metrics-server: :9965
   hubble-disable-tls: "false"
   hubble-listen-address: ""
-  hubble-socket-path: /dev/null
```

Comment:
Is this intentional?

Reply:
setting to /dev/null was for some brief testing, reverting

Reply:
@matmerr  I reverted it.